app/controllers/spree/paybright_controller.rb

```diff
@@ -63,13 +63,11 @@ def handle_callback_params!
       rescue StandardError => e
         logger.info "PAYBRIGHTDEBUG Order number: #{@payment.order.number}: Error: #{e.message}"
         logger.info "PAYBRIGHTDEBUG: Response code: #{@payment.response_code}"
+
         if @payment.response_code.present?
-          @payment.update_column(:state, :checkout) if @payment.invalid?
-          @payment.void
+          @payment.update(state: :completed) if !@payment.completed?
```

Comment:
They probably use `update_column` so it doesn't call any callbacks and go into an infinite loop, you might want to continue to use that.

Reply:
i wrote the `update_column` for checkout, bc you couldnt go from invalid -> checkout. but it should be OK to do this, i always use this method in console